examples/hello_pytorch/htcondor/apptainer/hello_pytorch_gpu_apptainer.sub

```diff
@@ -0,0 +1,44 @@
+# hello_pytorch_gpu_apptainer.sub
+# Submit file to access the GPU via apptainer
+
+# SIF file is located on /staging
+container_image = osdf:///chtc/staging/mfeickert/apptainer/pixi-docker-chtc.sif
```

Comment:
@aowen-uwmad As for Docker container use we have

https://github.com/UW-Madison-DSI/pixi-docker-chtc/blob/8961f77c33014adcd70a3a975e3228809d656788/examples/hello_pytorch/htcondor/hello_pytorch_gpu.sub#L4-L8

is it possible when you have Apptainer `.sif` container image files in a container registry like

```
oras://ghcr.io/uw-madison-dsi/pixi-docker-chtc:hello-pytorch-noble-cuda-12.9-apptainer
```

that we could directly provide the container image URI like we do for Docker? Or is there no way around: placing the container image on `/staging/`, copying it to the worker node, having to account for the container image and increase the memory and disk requirements on the worker?